config/300-eventing.yaml

```
@@ -71,6 +71,15 @@ spec:
                   URL:
                     description: The link of the manifest URL
                     type: string
+            additionalManifests:
+              description: A list of the additional eventing manifests, which will be installed by the operator
+              type: array
+              items:
+                type: object
+                properties:
+                  URL:
+                    description: The link of the additional manifest URL
+                    type: string
```

Just a suggestion but we could even declare an expected `format`, since OpenAPI supports it? I don't think it is taken into account during validation, but it's a nice hint I think. Up to you, feel free to reject the suggestion.
```suggestion
                    type: string
                    format: uri
```

---

We actually validate the resources, when using the manifestival library to retrieve them by the `spec.manifests`.
No need to worry about the format of this field.